src/components/App/App.js

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'font-awesome/css/font-awesome.min.css';
 import './App.css';
-import useHttpRequest from '../../hooks/useHttpRequest';
+import Home from '../Main/Home';
+import Header from '../Main/Header';
+import Login from '../Main/Login';
+import Error from '../Main/Error';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 
 function App () {
-  const {
-    state: { error, data, loading }
-  } = useHttpRequest({
-    method: 'GET',
-    url: '/users/62dd1bacf99611b5e71ec619'
-  });
-
   return (
-        <div className="App">
-            {error && <div>Error...</div>}
-            {loading && <div>Loading...</div>}
-            {data && (
-                <div>
-                    <div>id: {data._id}</div>
-                    <div>Name: {data.name}</div>
-                </div>
-            )}
-        </div>
+        <BrowserRouter>
+            <Header />
+            <main>
+                <Routes>
+                    <Route path="/" element={<Home />}></Route>
```

Comment:
If tag is empty, you can close it inside: <Route path="/" element={<Home />} />

Reply:
Fixed.